tests/system/Log/Handlers/FileHandlerTest.php

```diff
@@ -77,6 +78,100 @@ public function testHandleCreateFile()
         $this->assertStringContainsString($expectedResult, $line);
     }
 
+    public function testHandleCreateFilePHP()
+    {
+        $config                                                = new LoggerConfig();
+        $config->handlers[TestHandler::class]['path']          = $this->start;
+        $config->handlers[TestHandler::class]['fileExtension'] = 'php';
+        $logger                                                = new MockFileLogger($config->handlers[TestHandler::class]);
+
+        $logger->setDateFormat('Y-m-d');
+        $date     = date('Y-m-d');
+        $expected = 'log-' . $date . '.php';
+        $logger->handle('info', 'This is a test log');
```

Comment:
These codes are time-dependent.
If `date('Y-m-d')` is called on today 23:59:59.9999 and `$logger->handle()` is called on tomorrow 0:00:00,
This test fails.

Reply:
Yes, rare moment to get difference `ms`. If want real same time must give parameter to set name file manual.

Reply:
Now `FileHandler::handle()` is using `date()`. We should use `Time` instead of `date()`.
Then we can stop time during test execution.